.github/workflows/_test_workload.yml

```diff
@@ -42,8 +42,19 @@ jobs:
           az aks get-credentials --name $CLUSTER_NAME  --resource-group $RESOURCE_GROUP  --overwrite-existing
 
           echo "Wait for pod readiness before checking expected Kafka message."
-          kubectl wait --for=condition=Ready=true -n kafka pod/kafka-golang-consumer --timeout=180s
-          kubectl wait --for=condition=Ready=true -n kafka pod/kafka-producer --timeout=180s
+          kubectl wait --for=condition=Ready=true -n kafka pod/kafka-golang-consumer --timeout=180s || {
+            echo "Consumer is not ready within the acceptable time range. Printing out kubectl describe output."
+            result=$(kubectl describe pod -n kafka kafka-golang-consumer)
```

Comment:
is there any useful information that's logged? would using `kubectl logs` instead of (or in addition to) `kubectl describe` be helpful?

Reply:
Doing both could be good too.

Reply:
kubectl logs only shows workload logs, kubectl describe shoes pod events related logs.

Reply:
Also kubectl logs are printed in the if else statement